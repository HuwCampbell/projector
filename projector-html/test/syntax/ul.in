<ul>
  <li>
    Ham (<code>jamon</code>).
  </li>
  <li>
    Turkey (<code>turquia</code>).
  </li>
</ul>
