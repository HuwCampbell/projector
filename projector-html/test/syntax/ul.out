Template
  ()
  Nothing
  (TENode
     ()
     (THtml
        ()
        [ TElement
            ()
            (TTag () "ul")
            []
            (THtml
               ()
               [ TWhiteSpace () 1
               , TElement
                   ()
                   (TTag () "li")
                   []
                   (THtml
                      ()
                      [ TWhiteSpace () 1
                      , TPlain () TPlainText { unTPlainText = "Ham" }
                      , TWhiteSpace () 1
                      , TPlain () TPlainText { unTPlainText = "(" }
                      , TElement
                          ()
                          (TTag () "code")
                          []
                          (THtml () [ TPlain () TPlainText { unTPlainText = "jamon" } ])
                      , TPlain () TPlainText { unTPlainText = ")." }
                      , TWhiteSpace () 1
                      ])
               , TWhiteSpace () 1
               , TElement
                   ()
                   (TTag () "li")
                   []
                   (THtml
                      ()
                      [ TWhiteSpace () 1
                      , TPlain () TPlainText { unTPlainText = "Turkey" }
                      , TWhiteSpace () 1
                      , TPlain () TPlainText { unTPlainText = "(" }
                      , TElement
                          ()
                          (TTag () "code")
                          []
                          (THtml () [ TPlain () TPlainText { unTPlainText = "turquia" } ])
                      , TPlain () TPlainText { unTPlainText = ")." }
                      , TWhiteSpace () 1
                      ])
               , TWhiteSpace () 1
               ])
        ]))